File: Daily Challenge/Python/Flowchart Logic - 056 26.9.2025.py
M, N = map(int, input().split()); ctr = 1
while M > 0 or N > 0:
    if ctr % 2 != 0:
        print(str(N)[-1], end='')
        N = N // 10
    else:
        print(str(M)[-1], end='')
        M = M // 10
    ctr += 1

File: Daily Challenge/Java/Flowchart Logic - 056 26.9.2025.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int M = sc.nextInt(), N = sc.nextInt();
        int ctr = 1;
        while (M > 0 || N > 0) {
            if (ctr % 2 != 0) {
                System.out.print(N % 10);
                N /= 10;
            } else {
                System.out.print(M % 10);
                M /= 10;
            }
            ctr++;
        }
    }
}